apps/frontend/src/components/TimeCardPage/CellTypes/CommentCell.tsx

```diff
@@ -32,6 +46,21 @@ export function CommentCell({ comments, date, timesheetID }: CommentProps) {
     }
   }, [user?.Type]);
 
+  const updateReports = (updatedReports: ReportSchema[]) => {
+    setReports(updatedReports);
+
+    const reportsToComments = updatedReports.map((report) => ({
+      UUID: report.AuthorID,
+      AuthorID: report.AuthorID,
+      Type: report.Type,
+      Timestamp: report.Timestamp,
+      Content: `${report.Content},${report.Notified},${report.Explanation}`,
```

Comment:
is there a reason why we need to try and turn reports into comments? Can we just fully separate out the reports and comments in the fields?